google-cloud-kms/acceptance/google/cloud/kms/v1/smoke_test.rb

```diff
@@ -20,7 +20,7 @@
 class KmsServiceSmokeTest < Minitest::Test
   def test_list_key_rings
     kms = Google::Cloud::Kms.key_management_service version: :v1
-    key_ring_parent = kms.class.location_path ENV["KMS_PROJECT"], "us-central1"
+    key_ring_parent = kms.location_path project: ENV["KMS_PROJECT"], location: "us-central1"
```

Comment:
This change is leaking from the other PR. Revert it here to keep the PRs independent.

Reply:
oh shoot, sorry about that! I thought I took care of it.